modules/local/assignsh.nf

```diff
@@ -17,8 +17,6 @@ process ASSIGNSH {
     path outtable        , emit: tsv
     path "versions.yml"  , emit: versions
 
-    when:
-    task.ext.when == null || task.ext.when
```

Comment:
Just curious: Why are these deleted from local modules?

Reply:
Because its discouraged https://www.nextflow.io/docs/latest/strict-syntax.html#process-when-section and we never use it anyway.

Reply:
OK, good to know, thanks.